src/ApiService/ApiService/onefuzzlib/NotificationOperations.cs

```diff
@@ -134,6 +134,7 @@ public NotificationOperations(ILogger<NotificationOperations> log, IOnefuzzConte
         if (await _context.FeatureManagerSnapshot.IsEnabledAsync(FeatureFlagConstants.SemanticNotificationConfigValidation)) {
             var validConfig = await config.Validate();
             if (!validConfig.IsOk) {
+                _logTracer.LogError("Error(s) ocurred during template validation: {title}\r\n{errors}", validConfig.ErrorV.Title, string.Join("\r\n", validConfig.ErrorV.Errors ?? new()));
```

Comment:
You could use Environment.NewLine to get the right line endings all the time
https://learn.microsoft.com/en-us/dotnet/api/system.environment.newline?view=net-7.0